### DISCUSSION TITLE
Show placeholder characters always

### DISCUSSION THREAD

**Discussion by FirstVertex** [beholdr/maska#181] [Ideas]
Hi, I've been evaluating various masked input libraries for the last couple of days, and came across Maska, which looks promising.

The only issue is that I can't seem to get the placeholder characters to always be visible, even when the input is empty, or partially filled.  This is how our old library ([ngx-mask](https://github.com/JsDaddy/ngx-mask)) works (triggered by input of `{ showMaskTyped: true }` and I noticed [imask](https://github.com/uNmAnNeR/imaskjs) has a similar feature which is activated with `{ lazy: false }`.  It has also an input for `{ placeholderCharacter: ' ' }` which by default is `_` and I was able to customize to "space" to match our design system requirements.

An example would be, for a SSN input, it would always show the dashes, when empty, partially filled, or filled.
```txt
[   -  -    ]    <- empty
[123-4 -    ]    <- partial
[123-45-6789]    <- filled
```

If I somehow missed how this is triggered with the current api, please direct me to the doc for it, otherwise, thanks in advance for considering it as a new feature :v:

**Comment by jamiedoris** [comment]
I'm also looking for this feature -- migrating from vue-text-mask, which has a similar option.

**Comment by beholdr** [comment]
Implementing placeholder option for mask in pretty hard task, especially for dynamic and complex ones. I have no plans for such feature.
